### DISCUSSION TITLE
Azure private networking for GitHub-hosted runners

### DISCUSSION THREAD

**Discussion by pattisanta** [community/community#130783] [Other Feature Feedback, Questions, & Ideas]
Hi All,
I have setup all the requirements on the azure side to for github runners.  where I am stuck is  in github  when I go and add  azure virtual network,.  when I add Network settings resource ID. I don't know, what format its looking for? I have the resourceId from azure but it gives me an error when I try to add it. 
An unexpected error occurred: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

[Learn more on how to get the resource ID.](https://github.co/VNET_resourceID)

I look forward to any help I am completely stuck,
Patti

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @pattisanta!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the General category. I've gone ahead and moved it for you. Good luck!
